scripts/import/uniprot/uniprot_importer.py

```diff
@@ -5,18 +5,16 @@
 
 Options:
 
-    --g2p_host    G2P database host (Required)
-    --g2p_port    G2P database host port (Required)
-    --g2p_database    G2P database name (Required)
-    --g2p_user    G2P database Username (Required)
-    --g2p_password    G2P database Password (default: '') (Optional)
+    --config    Config file with details to the G2P database
 """
+
```

Comment:
Looks good, just wanted to suggest that we should add some typings

Reply:
Same thing here. This PR is only to add the necessary functionality to run the script this release.